### PR TITLE
Docs-11375 add runtime update topic

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+If needed, you can redeploy an application using an older runtime version. However, if you make any subsequent changes to that application, you must update its runtime version. 
+
+== Update a Single App to the Latest Runtime Patch Release
+
+Before updating apps in the production environment, update them in the sandbox environment and run required tests. 
+
+To update an app to the latest date patch:
+
+. In the Runtime Manager left menu, click *Applications*.
+. Under *Deployment Target*, from the *Runtime version* menu, select the new version.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. Under *Deployment Target*, from the *Runtime version* menu, select the version tagged `new`.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+If needed, you can redeploy an application using an older runtime version. However, if you make any subsequent changes to that application, you must update its runtime version. 
+
+== Update a Single App to the Latest Runtime Patch Release
+
+Before updating apps in the production environment, update them in the sandbox environment and run required tests. 
+
+To update an app to the latest date patch:
+
+. In the Runtime Manager left menu, click *Applications*.
+. Under *Deployment Target*, from the *Runtime version* menu, select the version tagged `new`.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. Under *Deployment Target*, from the *Runtime version* list, select the version tagged `new`.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+== Runtime Fabric Runtime Patch Update Schedule 
+
+Runtime Fabric runs each application and gateway deployment on a separate Mule runtime instance. By managing the containerized Mule images used for these deployments, Runtime Fabric enables you to upgrade your Mule runtime version for each deployment on the following schedule:
+
+. On the first full week of the month, you have access to new versions of updated Mule runtimes.
+. During the next two weeks, you can redeploy your applications using the newer runtime versions to validate its behavior. This is known as a grace period.
+. Beginning on the third full full week of the month, you must select the latest date patch version of your chosen Mule runtime version when deploying a new application, or when making a change to an existing application.
+
+[NOTE]
+If needed, you can redeploy an application using an older runtime version. However, if you make any subsequent changes to that application, you must update its runtime version.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
If needed, you can redeploy an application using an older runtime version. However, if you make any subsequent changes to that application, you must update its runtime version to the latest release. 
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.3.0:20210101`
+
+The monthly patches update applications only to the latest date patch for the patch version.
+They do not change the patch version number.
```
Comment (mcurrao-mulesoft) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: This is somewhat confusing. Perhaps we could say "They do not change the Runtime Version" instead?

Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
They do not change the runtime version.
```

Comment (IsaacEldridge) [reply modules/ROOT/pages/runtime-patch-updates.adoc]: Ok, fixed.

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+== Runtime Fabric Runtime Patch Update Schedule 
+
+Runtime Fabric runs each application and gateway deployment on a separate Mule runtime instance. By managing the containerized Mule images used for these deployments, Runtime Fabric enables you to upgrade your Mule runtime version for each deployment on the following schedule:
+
+. On the first full week of the month, you have access to new versions of updated Mule runtimes.
+. During the next two weeks, you can redeploy your applications using the newer runtime versions to validate its behavior. This is known as a grace period.
+. Beginning on the third full full week of the month, you must select the latest date patch version of your chosen Mule runtime version when deploying a new application, or when making a change to an existing application.
+
+[NOTE]
+If needed, you can redeploy an application using an older runtime version. However, if you make any subsequent changes to that application, you must update its runtime version to the latest release.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch version, and make any subsequent changes to that application, you may be required to update its date-patch version to a later release.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 6]
```diff
@@ -0,0 +1,55 @@
+= Mule Runtime Patch Updates for Runtime Fabric 
+
+The following provides important information about Mule runtime engine patch updates for Runtime Fabric.
+
+== Monthly Date-Patch Updates 
+
+Once per month, MuleSoft releases a _date-patch update_,  which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
Once per month, MuleSoft releases a _date-patch update_, which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+. If you want to see details about the version, click the *Read release notes* link.
+. Click the *Apply Changes* button to update the app. 
++
+Runtime Fabric updates the app with the version you selected and displays a message indicating success or failure.
+. Re-deploy your application.
++
+Mule deployments on Runtime Fabric are not automatically patched. You must re-deploy your applications to apply the latest date-patch updates.
+
+== See Also
+* Link to Best Practices with Mule Patching[]
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.3.0:20210101`
+
+The monthly patches update applications only to the latest date patch for the patch version.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: I'm confused by this sentence.

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+They do not change the runtime version.
+
+== Runtime Fabric Runtime Patch Update Schedule 
+
+Runtime Fabric runs each application and gateway deployment on a separate Mule runtime instance. By managing the containerized Mule images used for these deployments, Runtime Fabric enables you to upgrade your Mule runtime version for each deployment on the following schedule:
+
+. On the first full week of the month, you have access to new versions of updated Mule runtimes.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: suggest: "to new versions" --> ..to new date-patches"

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+. During the next two weeks, you can redeploy your applications using the newer runtime versions to validate its behavior. This is known as a grace period.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: suggest: "runtime versions" --> ..date-patch"

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+. Beginning on the third full full week of the month, you must select the latest date patch version of your chosen Mule runtime version when deploying a new application, or when making a change to an existing application.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: suggest omitting "version" when discussing "date patch version"

Comment (sundippatel) [reply modules/ROOT/pages/runtime-patch-updates.adoc]: suggest adding hyphen: "date patch" --> "date-patch"

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+[NOTE]
+During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch version, and make any subsequent changes to that application, you may be required to update its date-patch version to a later release.
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: suggest: "can select an earlier date-patch" (omit "version" when used after "date-patch").

File: modules/ROOT/pages/runtime-patch-updates.adoc (hunk 9 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. On the first full week of the month, you have access to new date-patches of updated Mule runtimes.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc (hunk 10 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. During the next two weeks, you can redeploy your applications using the newer date-patch to validate its behavior. This is known as a grace period.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc (hunk 11 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. Beginning on the third full full week of the month, you must select the latest date-patch of your chosen Mule runtime version when deploying a new application, or when making a change to an existing application.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc (hunk 12 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch, and make any subsequent changes to that application, you may be required to update its date-patch version to a later release.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc (hunk 8 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
The monthly patches update applications only to the latest date-patch. They do not change the runtime version.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.3.0:20210101`
+
+The monthly patches update applications only to the latest date-patch. They do not change the runtime version.
+They do not change the runtime version.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
```

**Review by sundippatel — DISMISSED** [review]

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 14]
```diff
@@ -0,0 +1,53 @@
+= Mule Runtime Patch Updates for Runtime Fabric 
+
+The following provides important information about Mule runtime engine patch updates for Runtime Fabric.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: First time in Level-1 topics is my own gen. rule:
```suggestion
The following provides important information about Mule runtime engine patch updates for Anypoint Runtime Fabric.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+
+`4.3.0:20210101`
+
+The monthly patches update applications only to the latest date-patch. They do not change the runtime version.
+
+== Runtime Fabric Runtime Patch Update Schedule 
+
+Runtime Fabric runs each application and gateway deployment on a separate Mule runtime instance. By managing the containerized Mule images used for these deployments, Runtime Fabric enables you to upgrade your Mule runtime version for each deployment on the following schedule:
+
+. On the first full week of the month, you have access to new date-patches of updated Mule runtimes.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. On the first full week of the month, you have access to new date-patches of updated Mule runtime instances.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 16]
```diff
@@ -0,0 +1,53 @@
...
+. During the next two weeks, you can redeploy your applications using the newer date-patch to validate its behavior. This is known as a grace period.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. During the next two weeks, you can redeploy your applications using the newer date-patch to validate its behavior. This is known as a _grace period_.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+. Beginning on the third full full week of the month, you must select the latest date-patch of your chosen Mule runtime version when deploying a new application, or when making a change to an existing application.
+
+[NOTE]
+During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch, and make any subsequent changes to that application, you may be required to update its date-patch version to a later release.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch, and make any subsequent changes to that application, you might be required to update its date-patch version to a later release.
```
"until they are no longer available" = ? deprecated? removed from <what>?

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+
+== Update a Single App to the Latest Runtime Patch Release
+
+Before updating apps in the production environment, update them in the sandbox environment and run required tests. 
+
+To update an app to the latest date patch:
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc:35]: Given the previous statement, is this and what follows AFTER the sandbox update and test?

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+
+. In the Runtime Manager left menu, click *Applications*.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: Is the "left menu" the "navigation menu"?

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 20]
```diff
@@ -0,0 +1,53 @@
...
+. Under *Deployment Target*, from the *Runtime version* list, select the version tagged `new`.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: colon pointing to image
```suggestion
. Under *Deployment Target*, from the *Runtime version* list, select the version tagged `new`:
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 21]
```diff
@@ -0,0 +1,53 @@
...
++
+image::rtf-runtime-select-version.png[The new runtime version is selected]
+Although the version number of the update might be the same as the installed version, a later date identifies the update:
++
+image::rtf-runtime-select-latest-version.png[The latest runtime version indicated by the date]
+. If you want to see details about the version, click the *Read release notes* link.
+. Click the *Apply Changes* button to update the app.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]: Just sayin', here and in most other cases, you can omit identifying the UI control type:
```suggestion
. Click *Apply Changes* to update the app. 
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
++
+image::rtf-runtime-select-version.png[The new runtime version is selected]
+Although the version number of the update might be the same as the installed version, a later date identifies the update:
++
+image::rtf-runtime-select-latest-version.png[The latest runtime version indicated by the date]
+. If you want to see details about the version, click the *Read release notes* link.
+. Click the *Apply Changes* button to update the app. 
++
+Runtime Fabric updates the app with the version you selected and displays a message indicating success or failure.
+. Re-deploy your application.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. Redeploy your application.
```

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 23]
```diff
@@ -0,0 +1,53 @@
...
++
+Mule deployments on Runtime Fabric are not automatically patched. You must re-deploy your applications to apply the latest date-patch updates.
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
Mule deployments on Runtime Fabric are not automatically patched. You must redeploy your applications to apply the latest date-patch updates.
```

**Review by kbostrom — DISMISSED** [review]
quick CE

File: modules/ROOT/pages/runtime-patch-updates.adoc [hunk 24]
```diff
@@ -0,0 +1,53 @@
...
+[NOTE]
+During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch, and make any subsequent changes to that application, you might be required to update its date-patch version to a later release.
+
+== Update a Single App to the Latest Runtime Patch Release
+
+Before updating apps in the production environment, update them in the sandbox environment and run required tests. 
+
+To update an app to the latest date patch:
+
+. In the Runtime Manager left menu, click *Applications*.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/runtime-patch-updates.adoc]:
```suggestion
. In the Runtime Manager navigation menu, click *Applications*.
```

**Review by hannanelson — APPROVED** [review]

**Review by sarjeet2013 — APPROVED** [review]

**Pull request merged**